mutiny-core/src/node.rs

```diff
@@ -795,6 +796,10 @@ impl Node {
             )
         };
 
+        self.persister
+            .storage
+            .set_invoice_labels(invoice.clone(), labels)?;
```

Comment:
Failing to save labels should not fail the payment that has already been attempted.

It would be nice if labels were just saved to the invoice db but probably too late at this point.